Cargo.toml

```diff
@@ -31,25 +23,19 @@ default = []
 # for quicker tests, cargo test --lib
 # for more explicit tests, cargo test --features=backtraces
 backtraces = ["cosmwasm-std/backtraces"]
-schema = ["cosmwasm-schema"]
 
 [dependencies]
-serde = { version = "1.0.147", default-features = false, features = ["derive"] }
-schemars = "0.8.11"
-cosmwasm-std = { version = "1.0.0" }
-cosmwasm-storage = { version = "1.0.0" }
-
-# Used only for schema generation
-cosmwasm-schema = { version = "1.1.0", optional = true }
+cosmwasm-std = { git = "https://github.com/scrtlabs/cosmwasm", tag = "v1.1.9-secret" }
+cosmwasm-storage = { git = "https://github.com/scrtlabs/cosmwasm", tag = "v1.1.9-secret" }
+schemars = { version = "0.8.11" }
+serde = { version = "1.0" }
+thiserror = { version = "1.0" }
+cosmwasm-schema = "1.0.0"
 
 # Uncomment these for some common extra tools
-# secret-toolkit = "0.7.0"
-# cw-storage-plus = { version = "0.14.0", default-features = false }
+# secret-toolkit = { git = "https://github.com/scrtlabs/secret-toolkit", tag = "v0.8.0" }
+# cw-storage-plus = { version = "1.0.1", default-features = false }
 
-[patch.crates-io]
-cosmwasm-std = { git = "https://github.com/scrtlabs/cosmwasm", rev = "f7e2b1dbf11e113e258d796288752503a5012367", package = "cosmwasm-std" }
-cosmwasm-storage = { git = "https://github.com/scrtlabs/cosmwasm", rev = "f7e2b1dbf11e113e258d796288752503a5012367", package = "cosmwasm-storage" }
 
-[[bin]]
-name = "schema"
-required-features = ["schema"]
\ No newline at end of file
+[patch.crates-io]
+cosmwasm-std = { git = "https://github.com/scrtlabs/cosmwasm", tag = "v1.1.9-secret" }
```

Comment:
This should be commented out too.
You don't need this if you're not using `cw-storage-plus` (or other cosmwasm-vanilla libs)